src/views/ArticleDetailPage.jsx

```diff
@@ -0,0 +1,273 @@
+            <p />
+            <p />
+            <p />
+            <p />
+            <p />
+            <p />
+          </article>
+        </div>
+        )}
+        { !isGetting && !errors.message && articleGotten.User && (
+        <div className="article_detail">
+          <div className="article_detail_header">
+            <img src={coverUrl} alt="article cover" className="article_detail_header_img" />
+            <section className="article_detail_header_info">
+              <h2 className="article_detail_header_info_title">
+                {title}
+              </h2>
+              <section className="article_detail_header_info_details">
+                <Link to={`/profiles/${User.username}`} className="article_detail_header_info_details_author">
+                  { (User.firstname || User.lastname) && `${User.firstname || ''} ${User.lastname || ''}`}
+                  { (!User.firstname && !User.lastname) && `${User.username}`}
+                </Link>
+                <span className="article_detail_header_info_details_date">
+                  {`${moment(createdAt).format('ha')} on ${moment(createdAt).format('MMMM Do YYYY')}` }
+                </span>
+                <span className="article_detail_header_info_details_time">
+                  <i className="fa fa-clock article_detail_header_info_details_time_img" />
+                  {readTime.text}
+                </span>
+              </section>
+              <section className="article_detail_header_info_tag">
+            Tag:
+                {' '}
+                <span className="article_detail_header_info_tag_span">{Tag.name}</span>
+              </section>
+            </section>
+          </div>
+          { isLoggedIn && (
+          <div className="article_detail_bookmark">
+            <i className="far fa-bookmark article_detail_bookmark_icon" />
+            <p>Add to Bookmark</p>
+          </div>
+          )}
+          <article className={`article_detail_body ${!isLoggedIn && 'article_detail_body_no_auth'}`}>
+            {body.split('<br />').map(section => (
+              <Fragment>
+                <article className="article_detail_body_segment">
+                  {section}
+                </article>
+                <br />
+              </Fragment>
+            ))}
+
+          </article>
+          <div className={`article_detail_rating ${!isLoggedIn && 'article_detail_rating_no_auth'}`}>
+            { isLoggedIn && (
+            <Fragment>
+              <p>How did you enjoy this article?</p>
+              <section className="article_detail_rating_stars">
+                <i className="far fa-star article_detail_rating_star" />
+                <i className="far fa-star article_detail_rating_star" />
+                <i className="far fa-star article_detail_rating_star" />
+                <i className="far fa-star article_detail_rating_star" />
+                <i className="far fa-star article_detail_rating_star" />
```

Comment:
Is there no way to present this icon from a loop, rather than adding it manually